far/viewer.cpp

```diff
@@ -296,13 +296,29 @@ wchar_t Viewer::ZeroChar() const
 
 void Viewer::ChangeHexModeBytesPerLine(int const Amount)
 {
-	const auto NewValue =
-		std::clamp(m_HexModeBytesPerLine + Amount, s_HexModeBytesPerStripe, hex_mode_bytes_from_line_width(ScrollbarAdjustedWidth, get_char_size(m_Codepage)));
+	if (Amount == 0) return;
 
-	if (NewValue == m_HexModeBytesPerLine)
-		return;
+	// Round up to the nearest multiple of AbsAmount.
+	const auto RoundUp = [](const auto BytesPerLine, const auto AbsAmount)
```

Comment:
RoundUp(32, 16) is 48, so this isn't actually "round up to the nearest multiple" (32 already is), more like "the next multiple". Maybe rename the function to better reflect what it does and avoid confusions with the math "round up" notation?

Reply:
Yeah! It makes sense. I renamed it to `GetNextGridPoint`. Will be happy to rename to something better if you suggest.